src/pudl/transform/eia861.py

```diff
@@ -648,6 +650,30 @@ def distribution_systems(raw_dfs, tfr_dfs):
         dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
 
     """
+    # No data tidying required
+    # Make sure numeric columns have no strings
+    logger.info('Transforming Distribution Systems table')
+    transformed_df = (
+        raw_dfs['distribution_systems_eia861'].copy()
+        .pipe(pudl.helpers.fix_eia_na)
+    )
+    # No duplicates to speak of but take measures to rid of them just in case
+    preduped_nrows = len(transformed_df)
+    transformed_df = transformed_df.drop_duplicates(
+        subset=['utility_id_eia', 'state', 'report_year'], keep=False)
+    deduped_nrows = len(transformed_df)
+    logger.info(
+        f"Dropped {preduped_nrows-deduped_nrows} duplicate records from EIA 861 "
+        f"distribution systems table, out of a total of {preduped_nrows} records "
+        f"({(preduped_nrows-deduped_nrows)/preduped_nrows:.4%} of all records). "
+    )
```

Comment:
If there are really zero duplicates, it might be better here check for duplicates, and raise an exception if any are found. E.g. something like
```py
dupes = transformed_df.duplicated(subset=["utility_id_eia", "state", "report_year"], keep=False)
if len(dupes) != 0:
    raise AssertionError(
        f"Found {len(dupes)} duplicate rows in the Demand Response table, when zero were expected!"
)
```
Alternatively, if there are a few duplicates which are just bad data, could you make a note in the comments or log message about what the nature of their badness is, and how many are expected?